ViewModel/ImpactAreaScenario/Results/Results.cs

```diff
@@ -1,17 +1,18 @@
-﻿using HEC.FDA.ViewModel.ImpactArea;
+﻿using HEC.FDA.ViewModel.Alternatives.Results;
+using HEC.FDA.ViewModel.ImpactArea;
 
 namespace HEC.FDA.ViewModel.ImpactAreaScenario.Results
 {
     public class Results
     {
         public DamageWithUncertaintyVM DamageWithUncertaintyVM { get; }
-        public DamageByDamageCategoryVM DamageByDamageCategoryVM { get; }
+        public DamageByDamCatVM DamageByDamageCategoryVM { get; }
         public PerformanceVMBase PerformanceAEPVM { get; }
         public PerformanceVMBase PerformanceAssuranceOfThresholdVM { get; }
         public PerformanceVMBase PerformanceLongTermRiskVM { get; }
-        public ImpactAreaRowItem ImpactArea { get; }
+        public ImpactArea.ImpactAreaRowItem ImpactArea { get; }
```

Comment:
Namespacing?

Reply:
it needs it because it is an ambiguous call.